Backend/TimeTracker.Api/Controllers/ProjectController.cs

```diff
@@ -161,6 +161,43 @@ public async Task<GenericResponseDTO<int>> AddTag(CreateTagDTO newTag)
             };
         }
 
+        /// <summary>
+        /// Adds a user to a project
+        /// </summary>
+        /// <param name="projectInvite">The invite code of the project that the user is added to</param>
+        /// <returns>The Id of the project the user was added to</returns>
+        [Authorize]
+        [HttpPost("AddUserToProject")]
+        public async Task<GenericResponseDTO<int>> AddUserToProject(string projectInvite)
```

Comment:
Post requests should have a DTO as their parameter. Right now this request is not working. I get `{"data":0,"message":"Couldn't find the project","success":false}` as a response each time even if it is a valid invite code.

Reply:
fixed